docs/document/website/versioned_docs/version-1.6.0/components/desktop/datetime-picker.md

```diff
@@ -0,0 +1,95 @@
+---
+id: version-1.6.0-datetime-picker
+title: DateTimePicker
+sidebar_label: DateTimePicker
+original_id: datetime-picker
+---
+
```

Comment:
Please create the same file with Japanese language at docs/document/website/translated_docs/ja/version-1.6.0